protocol/src/main/protos/core/Tron.proto

```diff
@@ -601,6 +601,7 @@ enum ReasonCode {
   TOO_MANY_PEERS_WITH_SAME_IP = 0x22;
   LIGHT_NODE_SYNC_FAIL = 0x23;
   BELOW_THAN_ME = 0X24;
+  NOT_WITNESS = 0x25;
```

Comment:
Is it compatible with older versions?

Reply:
All of the newly added unrecognized ReasonCode will be converted to UNKNOWN, and no exception will be thrown.

Reply:
@wubin01 there is no MESSAGE_WITH_WRONG_LENGTH really. But there is NO_SUCH_MESSAGE. Do you means add NO_SUCH_MESSAGE in Protocol.ReasonCode?

Reply:
I have added new ReasonCode NO_SUCH_MESSAGE in proto.